dqn.hpp

```diff
@@ -58,16 +73,16 @@ class DQN {
   void LoadTrainedModel(const std::string& model_file);
 
   // Restore solving from a solver file.
-  void RestoreSolver(const std::string& solver_file);
+  void RestoreSolver(const std::string& solver_actor_file);
```

Comment:
This method will now need to take two arguments: the solver for the critic and the solver for the actor.